src/pages/Sidebar.js

```diff
@@ -0,0 +1,156 @@
+        onClick={displayHideNavbar}
+      >
+        <HiMenu className="text-2xl" />
+      </button>
+      <div
+        className={
+          navHide
+            ? 'sidebar flex h-screen flex-col absolute justify-between border-r bg-white'
+            : 'sidebar-hidden flex h-screen flex-col absolute justify-between border-r'
+        }
+      >
+        <button
+          type="button"
+          className="close-sidebar sidebar-btn"
+          onClick={displayHideNavbar}
+        >
+          <RiCloseLine className="text-2xl" />
+        </button>
+        <div className="px-4 py-6 mt-6">
+          <span className="grid h-20 w-20  place-content-center text-center rounded-full bg-prime text-white">
+            Cherry Rooms
+          </span>
+
+          <nav aria-label="Main Nav" className="mt-6 flex flex-col space-y-1">
+            <NavLink
+              to="/rooms"
+              className="flex items-center gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillHome />
+              </span>
+              <span>Rooms</span>
+            </NavLink>
+
+            <NavLink
+              to="/reservations"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillDatabase />
+              </span>
+              <span>My Reservations</span>
+            </NavLink>
+
+            <NavLink
+              to="/reservation"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillBank />
+              </span>
+              <span>Reserve a Room</span>
+            </NavLink>
+            <NavLink
+              to="/rooms/create-room"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <FaSwatchbook />
+              </span>
+              <span>Create a Room</span>
+            </NavLink>
+
+            <NavLink
+              to="/delete"
+              className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            >
+              <span>
+                <AiFillDelete />
+              </span>
+              <span>Delete Room</span>
+            </NavLink>
+          </nav>
+        </div>
+
+        <div className="px-4 flex flex-col space-y-1">
+          <NavLink
+            to="/"
+            className="flex items-center cursor-pointer gap-2 rounded-lg hover:bg-prime hover:text-white px-4 py-2 text-gray-500"
+            onClick={handleClick}
+          >
+            <span>
+              <BiLogOutCircle />
+            </span>
+            <span>Logout</span>
```

Comment:
Can make navigation links to an array of objects and then map. like so
` const navs = [
{ to: '/',
navIcon: <BiLogOutCircle />,
navName: "Logout,
},
...
]

Then map out like so:`{navs.map((nav) => (
  <NavLink
    to={nav.to}
...
`